### PR TITLE
TopicManager JUnit test v1.0b

### PR DESCRIPTION
**Pull request by nicholas132000**

### CODE REVIEW

File: src/test/java/seedu/clialgo/TopicManagerTest.java [hunk 1]
```diff
@@ -104,4 +136,141 @@ void isValidTopic_invalidTopicInput_expectFalse() {
...
+
+    @Test
+    void isTopicEmpty_givenTopicIsNotEmpty_expectFalse() {
+        TopicManager topicManager = new TopicManager();
+        String noteName = "queue";
+        String topicName = "LINKED_LIST";
+        Note note = new Note(noteName, "dummy path", topicName);
+        topicManager.addNote(noteName, topicName, note);
+        assertFalse(topicManager.isTopicEmpty(topicName));
+    }
```
Comment (heejet) [inline src/test/java/seedu/clialgo/TopicManagerTest.java:164]: What is the difference between this and lines 141 to 148?

File: src/test/java/seedu/clialgo/TopicManagerTest.java [hunk 2]
```diff
@@ -104,4 +136,141 @@ void isValidTopic_invalidTopicInput_expectFalse() {
...
+        assertEquals(expectedOutcomeForLinkedList, topicManager.getNotesByTopic("LINKED_LIST"));
+
+        ArrayList<String> expectedOutcomeForSorting = new ArrayList<>();
+        expectedOutcomeForSorting.add("bubble sort");
+        expectedOutcomeForSorting.add("merge sort");
+        assertEquals(expectedOutcomeForSorting, topicManager.getNotesByTopic("SORTING"));
+    }
+
+    @Test
+    void initialize_xx() {
```
Comment (heejet) [inline src/test/java/seedu/clialgo/TopicManagerTest.java:253]: is the xx part of the coding standard?

**Review by heejet — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/clialgo/TopicManagerTest.java (hunk 1 above)
Comment (nicholas132000) [reply src/test/java/seedu/clialgo/TopicManagerTest.java:164]: 141-148 is the test to check for whether allNotes in topicManager is empty, whereas 157-158 checks for whether a particular topic is empty

File: src/test/java/seedu/clialgo/TopicManagerTest.java (hunk 2 above)
Comment (nicholas132000) [reply src/test/java/seedu/clialgo/TopicManagerTest.java:253]: No, thanks for spotting, will amend it with the appropriate coding standard.

**Review by nicholas132000 — COMMENTED** [review]
Replied to the feedback, will implement the necessary changes

File: src/test/java/seedu/clialgo/TopicManagerTest.java (hunk 1 above)
Comment (heejet) [reply src/test/java/seedu/clialgo/TopicManagerTest.java:164]: Ah I get it. Thanks!

**Review by heejet — APPROVED** [review]

**Pull request merged**